### ISSUE TITLE
Notes and Examples should not allow empty lines at block start/end

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod-utils#5]
e.g.

```
[NOTE]
--

A physical quantity instance is a class that a physical thing can be a member of. Whether or not a physical thing is a member of the class is determined by the result of an observation or measurement.

A physical quantity space is a class that a physical quantity instance can be a member of. Whether or not a physical quantity instance is a member of the class is determined by the nature of the physical phenomenon that is observed or measured.

--
```

Originally from: metanorma/reverse_adoc#47

**Comment by ronaldtse** [comment]
Still not fixed in #16:

```adoc
[NOTE]
--

Various equalities relating topological properties of entities are derived from the invariance of a number known as the Euler characteristic. Typically, these are used as quick checks on the integrity of the topological structure. A violation of an Euler condition signals an "impossible" object. Two special cases are important in this document. The Euler equation for graphs is discussed in 5.2.3. Euler conditions for surfaces are discussed in 5.4.25 and 5.2,27.

--
```

**Comment by ronaldtse** [comment]
Fixed.
